tests/test_postgresql.py

```diff
@@ -1258,6 +1258,30 @@ def test_inline_exclude_constraint_text(self):
             "name='TExclID'))",
         )
 
+    def test_drop_exclude_constraint(self):
```

Comment:
move this test to tests/test_autogen_composition.py

Reply:
Are you sure that `test_autogen_composition.py` is a better location?  Other tests for Postgres exclusion constraints are all in `test_postgresql.py` from what I can see (e.g. `test_create_exclude_constraint`).

Reply:
while this is using `ExcludeConstraint` , the idea is that it would work for *any* constraint object coming from any third party dialect.    Ideally we'd test both `ExcludeConstriant` in the pg tests as well as some kind of generic "XYZConstraint" here.

Reply:
Understood.  I'm new to this project and don't quite see how to easily change test_autogen_composition tests.  If you wish to point me in the right direction and/or make a change for me, please do.

Reply:
Thanks.